components/resource-creation/ResourceDisplay.tsx

```diff
@@ -129,37 +119,20 @@ function ResourceDisplay() {
             }
             // Else if dataType === 'dateTime' || 'Date'
             else {
-              return (
-                <Tooltip
-                  arrowPosition="side"
-                  arrowOffset={25}
-                  arrowSize={8}
-                  label={fullResourceDateName}
-                  withArrow
-                  position="top-start"
-                >
-                  <Text>{formatDate(fhirpath.evaluate(resource, fullResourceDateName)[0])}</Text>
-                </Tooltip>
-              );
+              return [formatDate(fhirpath.evaluate(resource, fullResourceDateName)[0]), fullResourceDateName];
```

Comment:
'Text' and 'Tooltip' can be removed from the top (see lint error warning when running `npm run check`)